src/controllers/photo.ts

```diff
@@ -39,6 +39,12 @@ const getAll = async (request: ExtendedFastifyRequest, reply: FastifyReply) => {
 
 const post = async (request: ExtendedFastifyRequest, reply: FastifyReply) => {
   try {
+    const credentials: any = request.user
```

Comment:
You use the same code for the user checking at 3 different places , U can maybe try to create an helper function or something like that to avoid code rewriting 😁

Reply:
I don't think it's really important for now, especially since conditions may change depending on the context. We'll improve it later (seen together)